pygmt/src/contour.py

```diff
@@ -79,10 +79,11 @@ def contour(self, x=None, y=None, z=None, data=None, **kwargs):
         Do NOT clip contours or image at the boundaries [Default will clip
```

Comment:
I can't comment on line 62 since you didn't make changes there, but please format the "A" doc string arguments and also include it on the list of aliases.

Reply:
Yes, this is a separate issue with **A**, since it's not just a formatting problem. I will submit a separate PR that addresses [issue 879](https://github.com/GenericMappingTools/pygmt/issues/879) in a separate PR.

Reply:
Fixed in #883